### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by CorneliusCornbread** [yessGlory17/programmer-browser#42] [Ideas]
I'm not sure how the adblock is done but I assume it's some integration of ublock, so why not have a similar integration with Dark Reader? It'd allow for the programmer browser to be dark mod anywhere so I don't have to look at a Dark Mode IDE and a light mode browser at the same time.

**Comment by yessGlory17** [comment]
We are currently in alpha version. We will be supporting all chrome extensions when we switch to beta version. If you like and useful add-ons, add their links as comments. It also comes with adblock embedded. You can see that it blocks Youtube ads.
